app/models/update_certificate_notification.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateCertificateNotification < AggregatedEvent
+  belongs_to_aggregate :certificate
+  data_attributes :notification_sent
+
+  def attributes_to_apply
+    { notification_sent: notification_sent || false }
```

Comment:
```suggestion
    { notification_sent: true }
```
Again, no need to pass anything in, this event should be fired when a notification is sent